src/components/UI/DropDown/DropDown.tsx

```diff
@@ -0,0 +1,36 @@
+import { ChangeEvent, useState } from 'react'
+import classes from './DropDown.module.css'
+
+const DropDown = ({
+  choices,
+  onChange,
+}: {
+  choices: Array<string>
+  onChange?: (event: string) => void
+}) => {
+  const [track, setTrack] = useState(choices[0])
+
+  const onChangeHandler = (event: ChangeEvent<HTMLSelectElement>) => {
+    setTrack(event.target.value)
+    if (onChange) {
+      onChange(event.target.value)
+    }
+  }
+
+  let key = 0
```

Comment:
`key` will be reset to 0 on every component re-render. This is not a solution. Instead, you can use `choice` as this:
```tsx
<option key={choice}>{choice}</option>
```
Since `choice` should be unique strings.